Dnn.AdminExperience/ClientSide/Dnn.React.Common/src/Tags/index.jsx

```diff
@@ -185,8 +185,8 @@ Tags.propTypes = {
 };
 
 Tags.defaultProps = {
-    addTagsPlaceholder: "Add Tags",
-    searchTagsPlaceholder: "Begin typing to search tags",
+    addTagsPlaceholder: {Localization.get("addTagsPlaceholder")},
+    searchTagsPlaceholder: {Localization.get("searchTagsPlaceholder")},
```

Comment:
I am not sure code can run in defaultProps definition, I am not 100% sure but the build is failing